build/cpu.py

```diff
@@ -43,6 +43,11 @@ class IA64(CPU):
 	'''
 	name = 'ia64'
 
+class LOONGARCH64(CPU):
+   '''64-bit LOONGARCH64.
+   '''
+   name = 'loongarch64'
```

Comment:
It seems the indentation for these lines is different from the rest of the file. Please indent with one tab.

(While 4 spaces is more common in Python, tabs are what the C++ part of openMSX uses and we stuck with that for internal consistency.)

Reply:
Sure, I'll make some modifications. Thank you for your suggestions.

Reply:
Thanks!